app/javascript/stylesheets/application.scss

```diff
@@ -1,6 +1,25 @@
 @import "~bootstrap/scss/bootstrap";
 @import "./_sidebar";
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
 
 #page-content-wrapper {
   width: 100%;
 }
+
+.big-button {
+  width: 100%;
+}
+
+.column-background {
+  display: flex;
+  flex-direction: column;
+  object-fit: contain;
+  background-image: linear-gradient(103deg, #222E62, #6A98CC);
+  background-repeat: no-repeat;
+  background-size: cover;
+  align-items: center;
+  justify-content: center;
```

Comment:
Some of the styling that you added here can also be set by using pseudoclasses (e.g. 'justify-content-center'). I'd suggest taking each of the attributes here and checking if there is a Bootstrap class that you could use and set directly on the element.

Reply:
I added possible pseudo classes.